extension_templates/annotation.py

```diff
@@ -179,3 +183,30 @@ def _update(self, X, Y=None):
 
         # implement here
         # IMPORTANT: avoid side effects to X, fh
+
+    # todo: return default parameters, so that a test instance can be created
+    @classmethod
+    def get_test_params(cls):
+        """Return testing parameter settings for the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, (i.e),
```

Comment:
remove brackets around `i.e.`, line should be 
`Each dict are parameters to construct an "interesting" test instance, i.e.,`